sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -528,7 +538,7 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
   case PI_DEVICE_INFO_VERSION:
     // CM_EMU release version from
     // https://github.com/intel/cm-cpu-emulation/releases
-    return ReturnValue("1.0.7-CM_EMU");
+    return ReturnValue("1.0");
```

Comment:
this should rather be a query into libcm. at least TODO should be added

Reply:
I'll check if the version number can be retrieved from CM and apply it if possible.

And, please note that the device info version format should be 'X.Y'. It is required from one of tests in intel/llvm-test-suite.